### PR TITLE
docs: Mention updating Helm Charts for the first time configuration

### PR DESCRIPTION
**Pull request by jjerphan**
Hi,

First, thank you for developing Substra!

I am trying setting up the development environment given in the documentation but I have had problems when running `skaffold run`.

Those steps might be needed in order to resolve the charts for `substra/orchestrator` and `substra/substra-backend`.

cc @Fabien-GELUS, whom I mentioned this.

On a side note: I have opened this Pull Request without opening any issue before hand. I searched for contributing conventions and guidelines (usually given in the doc or in `CONTRIBUTING.{md,est}`) but haven't been able to. Do you have any, yet? Thank you!

Signed-off-by: Julien Jerphanion <git@jjerphan.xyz>

### CODE REVIEW

**Comment by ThibaultFy** [comment]
Thanks for the PR! 
Just to answer for the contribution part, we are currently working on the contributing guide (cf [here](https://github.com/Substra/substra-documentation/pull/230) ). It will be available ASAP !

**Comment by jjerphan** [comment]
Thanks, @ThibaultFy!

**Comment by guilhem-barthes** [comment]
Hi @jjerphan, thanks for your submission! Would you have more context to share with us about this issue? We didn't manage to reproduce your problem.

**Comment by jjerphan** [comment]
Hi @guilhem-barthes, charts were not found when I was running `skaffold` commands. The commands that I added makes the charts accessible.

I can't reproduce the error (because those charts now exist), but I might flush caches or so so that I can.

**Comment by jjerphan** [comment]
@ThibaultFy and @guilhem-barthes: I offer no guaranty that I can reach the problem I was facing, yet I can try coming up with a reproducer for it ; would you be interested?

**Comment by ThibaultFy** [comment]
Hi @jjerphan ! Thanks for proposing your help, but we are currently deploying Substra from scratch on several machine in order to reproduce the issue you pointed out. We'll be able to come back at you and work on your contribution very soon normally 😄 !

File: docs/source/contributing/getting-started.rst [hunk 1]
```diff
@@ -151,6 +151,17 @@ First time configuration
...
       npm install --dev
 
+7. Update Helm charts
+
+   .. code-block:: bash
+
+      pushd orchestrator/charts/orchestrator/
+      helm dependency update
+      popd
+      pushd substra-backend/charts/substra-backend/
```
Comment (guilhem-barthes) [inline docs/source/contributing/getting-started.rst:161]:
```suggestion
      cd substra-backend/charts/substra-backend/
```

File: docs/source/contributing/getting-started.rst [hunk 2]
```diff
@@ -151,6 +151,17 @@ First time configuration
...
+      helm dependency update
+      popd
```
Comment (guilhem-barthes) [inline docs/source/contributing/getting-started.rst:163]:
```suggestion
      cd ../../../
```

**Review by guilhem-barthes — CHANGES_REQUESTED** [review]
Hi @jjerphan, thanks for your your contribution and your patience. I am sorry we took so much time to consider and review this contribution, we had some internal deadlines to meet which slowed down this process. We are able to reproduce and to also notice that your changes where needed in a fresh install. 

As you can see in the suggested changes, we do not want to introduce the tools `pushd` and `popd` as it is not as well-known as `cd` and may create more confusion on first-time user. If you want me to apply the suggested changes, please let me know!

**Comment by jjerphan** [comment]
Hi @guilhem-barthes, no problem for the delay: I was not necessarily expecting a prompt response. :slightly_smiling_face: 

I have committed your suggestions to use `cd`.

**Review by guilhem-barthes — APPROVED** [review]

**Pull request merged**